ONSdigital/dp-frontend-filter-dataset-controller
# Time refactor

### What

- Added time handlers and mappings to specifically handle time dimension
- Remove reference of time in generic dimension handlers

### How to review

- Create a filter job on the frontend web journey
- Click on time

Verify that:
 - You can add only the latest time
 - You can can add only a single time
 - You can add a time range
 - You can add a list of times
 - Your most recent selection overrides any previous selection

### Who can review

Team A
